README.md

```diff
@@ -0,0 +1,99 @@
+# WSDL Docs
+SOAP API documentation via WSDL imports.  Converts operations from the WSDL to nodes.
+
+## SETUP
+* This module creates nodes from Operations in a WSDL and allows sending of SOAP requests for those operations.
+* Ensure SOAP is installed on your server.
+* Enable Required modules
+    * wsdl_docs
+    * wsclient
+    * wsclient_soap
+    * entityreference
+    * pathauto (If you want SEO friendly URLs on operation nodes)
+    * services and rest_server (If you want to create a REST endpoint for CRUD operations on WSDL docs)
+* Enable Features
+    * wsdl_docs_features
+* Permissions: There is a 'WSDL Docs Operations using Services module' permission that is checked when making requests via REST. Make sure you add a user that has this permission, and use that user's login credentials when making REST requests.
+
+## USAGE
+* Import WSDL via Drupal admin UI
+    * /admin/content/wsdl_docs/add
+* View created operation nodes
+    * /admin/content, filter by Type: 'WSDL Docs Operation'
+* Update existing service/operations via Drupal admin UI
+    * Click edit on admin/content/wsdl_docs for the service you want to update the operations on.
+    * Click save.
+
+### CUD WSDL Docs via REST (thanks to Services module)
+1. Make sure the services module is enabled
+1. At admin/structure/services/add add your service, select REST as the Server, and save. Select HTTP basic authentication for the authentication.
+1. Once you've added your service, click on the Edit Resources configuration option at admin/structure/services for your service
+1. Under the WSDL resource, select the operations you want to make available (CUD available only right now) and save
+
+### cURL commands for the operations enabled by this module:
+Note: Use HTTP Basic Auth for these APIs (Username/password is the credentials for the account with 'WSDL Docs Operations using Services module' permissions setup on DevPortal)*
+#### Create a WSDL Doc Service
+`curl -X POST [your-website-url]/[your-service-path-to-endpoint]/wsdl \
+-H 'Content-Type: application/json' \
+-H 'Authorization: Basic c2VydmljZXMtYWRtaW46cGFzc3dvcmQ=' \
+-d '{"name": "test8", "url": "http://www.thomas-bayer.com/axis2/services/BLZService?wsdl"}'`
+#### Update a WSDL Doc Service's Operations by URL of WSDL resource
+`curl -X PUT [your-website-url]/[your-service-path-to-endpoint]/wsdl/[wsdl-doc-name] \
+-H 'Authorization: Basic c2VydmljZXMtYWRtaW46cGFzc3dvcmQ=' \
+-H 'Content-Type: application/json' \
+-d '{"url": "https://graphical.weather.gov/xml/DWMLgen/wsdl/ndfdXML.wsdl"}'`
+#### Delete a WSDL Doc Service and its Operations
+`curl -X DELETE [your-website-url]/[your-service-path-to-endpoint]/wsdl/[wsdl doc name] \
+-H 'Authorization: Basic c2VydmljZXMtYWRtaW46cGFzc3dvcmQ='`
+#### Import a WSDL to an Existing Service by WSDL file
```

Comment:
add a section that describes the 2 ways you can get your wsdl into devportal:
1. call create, then call update with a url OR
2. call create, then call import with a file